metadata/focus-change.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
```

Comment:
You need to install this xml file by adding it in metadata/meson.build like the others.